sdk/documentintelligence/azure-ai-documentintelligence/CHANGELOG.md

```diff
@@ -3,12 +3,40 @@
 ## 1.0.0b4 (Unreleased)
 
 ### Features Added
+- Added support for the Analyze Batch Documents API:
+    - Added LRO operation `begin_analyze_batch_documents()` to `DocumentIntelligenceClient`.
+    - Added models `AnalyzeBatchDocumentsRequest`, `AnalyzeBatchResult` and `AnalyzeBatchOperationDetail`.
+- Added support for different kinds of output in the Analyze Document API:
+    - Added operations `get_analyze_result_figure()` and `get_analyze_result_pdf()` to `DocumentIntelligenceClient`.
+    - Added optional kwarg `output` to LRO operation `begin_analyze_document()` overloads in `DocumentIntelligenceClient`.
+    - Added enum `AnalyzeOutputOption` to specify output kind, either `pdf` or `figures`.
+    - Added property `id` to model `DocumentFigure`.
+- Added support for the Copy Classifier API:
+    - Added operations `authorize_classifier_copy()` and `begin_copy_classifier_to()` to `DocumentIntelligenceAdministrationClient`.
+    - Added models `AuthorizeClassifierCopyRequest` and `ClassifierCopyAuthorization`.
+- Added optional kwarg `pages` to LRO operation `begin_classify_document()` overloads in `DocumentIntelligenceClient`.
+- Added new kind `GENERATIVE` to enum `DocumentBuildMode`.
+- Added property `warnings` to model `AnalyzeResult`.
+- Added properties `classifier_id`, `split`, and `training_hours` to model `DocumentModelDetails`.
+- Added properties `model_id`, `confidence_threshold`, `features`, `query_fields` and `max_documents_to_analyze` to model `DocumentTypeDetails`.
+- Added property `allow_overwrite` to model `BuildDocumentClassifierRequest`.
+- Added properties `allow_overwrite` and `max_training_hours` to model `BuildDocumentModelRequest`.
+- Added properties `classifier_id`, `split` and `doc_types` to model `ComposeDocumentModelRequest`.
 
 ### Breaking Changes
-
-### Bugs Fixed
+- Removed support for extracting lists from analyzed documents:
+    - Removed models `DocumentList` and `DocumentListItem`.
```

Comment:
just checking - are there any replacements that should be used instead or these are simply removed and is no longer available?

Reply:
These two models were used in AnalyzeResult property which are removed in new API.